Sources/OpenAIAPIManager/OpenAIAPIManager.swift

```diff
@@ -177,7 +177,7 @@ final public class ChatGPTAPIManager {
 
     
     // MARK: - Private Functions -
-    private  func textEditsRequest(endPoint: ChatGPTAPIEndpoint, model: EditGPTModels, input: String?, instruction: String, n: Int, temperature: Double, topP: Double, completion: @escaping (Result<[String],Error>) -> Void) {
+    private  func textEditsRequest(endPoint: OpenAIAPIManagerEndpoints, model: EditGPTModels, input: String?, instruction: String, n: Int, temperature: Double, topP: Double, completion: @escaping (Result<[String],Error>) -> Void) {
```

Comment:
The new name of endpoints class should be OpenAIAPIEndpoints instead of OpenAIAPIManagerEndpoints.